packages/now-cli/src/util/output/capitalize.ts

```diff
@@ -0,0 +1,3 @@
+export default function capitalize(str: string) {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
```

Comment:
We already have a dependency that does this called `title`.

See usage here: https://github.com/vercel/vercel/blob/1bfa310945831e9d1204122557bae16d0c6bfe5a/packages/now-cli/src/util/pkg-name.ts#L21-L24

Reply:
TIL! Updated in https://github.com/vercel/vercel/pull/5372/commits/06d0a6f8296b938642e4a5d2a562321708434a8f 👍